Lib/test/mapping_tests.py

```diff
@@ -622,6 +622,7 @@ def __repr__(self):
         d = self._full_mapping({1: BadRepr()})
         self.assertRaises(Exc, repr, d)
 
+    @skip_emscripten_stack_overflow
```

Comment:
I think this is the cause of the CI test failures
```suggestion
    @skip_emscripten_stack_overflow()
```

Reply:
Thanks!